### ISSUE TITLE
NCPA hang if a nfs mount is stalled

### ISSUE THREAD

**Issue by metabsd** [NagiosEnterprises/ncpa#368]
All checks for this server are critical. The problem is about a NFS share appear to be stalled.

Do you thing you can try/catch or handle that exception and continue to send status of all other check.
That problem on NFS generate a lot of Critical on the server and CPU Usage/ Memory etc.. are critical.

```
2017-08-21 09:03:33,033 2284 ERROR Exception on /api/disk/logical/|var|log/ [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
  File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
  File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
  File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
  File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
  File "/root/ncpa/agent/listener/server.py", line 154, in token_auth_decoration
  File "/root/ncpa/agent/listener/server.py", line 920, in api
  File "/root/ncpa/agent/listener/psapi.py", line 237, in getter
  File "/root/ncpa/agent/listener/psapi.py", line 184, in get_root_node
  File "/root/ncpa/agent/listener/psapi.py", line 153, in get_disk_node
  File "/root/ncpa/agent/listener/psapi.py", line 59, in make_mountpoint_nodes
OSError: [Errno 5] Input/output error: '/opt/app/ce_externe_qlf_90/logs/slqwas0613'
```

Thank you!

**Comment by jomann09** [comment]
Adding this for me. Will catch the OSError in 2.0.x and in 2.1 not only will it catch the error but by default NFS mounts will be disabled because of enhancement #328

**Comment by jomann09** [comment]
I've added this in c2d9946059f8e4c5f223906573bdcf72a8774e29 and like I stated above, the excludes list will be available in 2.1!
